DEPLOYMENT_CHECKLIST.md

```diff
@@ -38,6 +38,7 @@ To deploy run `yarn deploy --network NETWORK` in `packages/contracts` and replac
 - [ ] Update `active_contracts.json` with the new deployed addresses
 - [ ] Update `packages/contracts/Releases.md` with the new deployed addresses
 - [ ] Add the managing DAOs' multisig address to `packages/contracts/.env.example` in the format `{NETWORK}_MANAGINGDAO_MULTISIG`
+- [ ] Add a Github Release with the version number as tag and the defined content (see Notion)
```

Comment:
Not clear which notion. Once we have the first release, everyone can use this one as a reference for how it should look like. Accordingly, I would remove "and the defined content (see Notion)".

Reply:
better?